apps/dashboard_web/lib/dashboard_web/live/components/check_in_count.ex

```diff
@@ -0,0 +1,42 @@
+defmodule DashboardWeb.Components.CheckInCount do
+  use DashboardWeb, :live_component
+
```

Comment:
Since you've already merged my PR, I'm going to make some suggestions based on work done there

```suggestion
@behaviour DashboardWeb.Behaviours.ComponentLiveView
```